packages/sources/src/components/card/card.models.ts

```diff
@@ -6,4 +6,5 @@ export interface Card<TemplateFnReturnType = never> {
   selectable?: Selectable<TemplateFnReturnType>;
   content: string;
   interactions?: Button[];
+  imgSrc?: string;
```

Comment:
imgSrc zegt niets over de toegevoegde functionaliteit. Ik zie dat de story het ook gewoon heeft over "Afbeelding", maar laten we dan wegblijven van de HTML termen..

Ik denk dat we gewoon voor `image?: string` moeten gaan

Reply:
Mee eens, ik pas het aan